ui/DesignSystem/Component/SourceBrowser/CommitTeaser.svelte

```diff
@@ -59,7 +60,7 @@
     <Icon.Commit style="fill: var(--color-secondary)" />
     <a
       class="commit-sha"
-      href={path.projectCommit(projectId, commitSha)}
+      href={path.projectCommit(projectId, userId, commitSha)}
```

Comment:
Why does this depend on user information?

Reply:
I think it's the `PeerId`?

Reply:
Yeah, it'll be the `PeerId` for now, but I thought that we'd be able to resolve them to user's soon, is that not correct?

Reply:
@cloudhead addressed in https://github.com/radicle-dev/radicle-upstream/pull/620/commits/d5b10c171dddbf5b93c4a3b1493d25c1689799f0

@FintanH no, this just constructs a link to the single commit screen, when you click the link the screen will fetch the respective commit and use these params as scope.

<img width="50%" alt="Screenshot 2020-07-01 at 16 21 40" src="https://user-images.githubusercontent.com/158411/86255056-f8c01000-bbb6-11ea-81a4-0ba60a2673d0.png">

Reply:
> I think it is fine to be able to jump to any commit, as long as some metadata is computed which shows in which projects + branches it appears. The only ambiguous case is when you jump to a commit in the current project, but that commit doesn't appear in that project anywhere.

As Kim said, it might not be that we scope it. I would suggest we remove this change until we're sure what the behaviour will be.

Reply:
Alright.

Reply:
@FintanH https://github.com/radicle-dev/radicle-upstream/pull/620/commits/cc7bc8a3819ab9ab419487c80f0758ca56898462